### PR TITLE
replaced all instances of keizer-auth-api with keizer auth

### PR DESCRIPTION
**Pull request by sanatan-dive**
# Changes Needed

  -  Update go.mod file to rename module from keizer-auth-api to keizer-auth
  - Update all internal import paths to reflect the new module name
  -  Update any CI/CD configurations referencing the old module name
  - Update documentation and README files

# Changes Done
## updated all the instances of keizer-auth-api to keizer-auth
  _-  Updated go.mod file to rename module from keizer-auth-api to keizer-auth_
  _- Updated all internal import paths to reflect the new module name_
  _-  Updated any CI/CD configurations referencing the old module name_
  _- Updated documentation and README files_

### CODE REVIEW

**Comment by sanatan-dive** [comment]
i think i have replaced all the instances of auth-api to just auth and i guess everything should be fine still if any issues persists let me know , also i dont have any experience with go expect running "main.go" files

**Comment by Sudarsh1010** [comment]
> i think i have replaced all the instances of auth-api to just auth and i guess everything should be fine still if any issues persists let me know , also i dont have any experience with go expect running "main.go" files

Thanks for the Contribution. can you provide build logs? If it's build's fine LTGM

**Comment by sanatan-dive** [comment]
can you please provide me steps to access build logs, when searching on google it shows this commands
```
go list -m all
```
is it correct or do i have to run something else

**Comment by Sudarsh1010** [comment]
> can you please provide me steps to access build logs, when searching on google it shows this commands
> 
> ```
> go list -m all
> ```
> 
> is it correct or do i have to run something else

my bad, I meant screenshot of successful build
```make build```

File: web/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5912 @@
+{
+  "name": "web",
+  "version": "0.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "web",
+      "version": "0.0.0",
+      "dependencies": {
```
Comment (Sudarsh1010) [inline web/package-lock.json:10]: and remove package-lock.json, we make use pnpm. README is not updated yet. so please remove package-lock.json

**Comment by sanatan-dive** [comment]
i removed package-lock.json and i think readme.md is updated also heres the screenshot

Screenshot_10-Nov_00-14-44_code

**Review by Sudarsh1010 — APPROVED** [review]

**Comment by Sudarsh1010** [comment]
make sure to link issue from next PR, thanks for the contribution

resolves #20

**Comment by sanatan-dive** [comment]
oh mb ill remember from next time thank you!

**Comment by Sudarsh1010** [comment]
> oh mb ill remember from next time thank you!

there are bunch of good first issue. please do have a chat on the issue thread that you interested to work. so If we any input that you may find useful we will provide you and you will be assigned to it. Thanks

**Comment by sanatan-dive** [comment]
sure thanks

**Pull request merged**